shared/src/commonMain/kotlin/dev/johnoreilly/confetti/utils/DateService.kt

```diff
@@ -1,11 +1,26 @@
 package dev.johnoreilly.confetti.utils
 
-import kotlinx.datetime.Instant
+import kotlinx.coroutines.currentCoroutineContext
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.isActive
 import kotlinx.datetime.LocalDateTime
 import kotlinx.datetime.TimeZone
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
 
 interface DateService {
     fun format(localDateTime: LocalDateTime, timeZone: TimeZone, format: String): String
 
     fun now(): LocalDateTime
-}
\ No newline at end of file
+}
+
+fun DateService.createCurrentLocalDateTimeFlow(delay: Duration = 5.minutes): Flow<LocalDateTime> =
```

Comment:
@marcellogalhardo I think this should be in the TZ of the conference, otherwise the comparisons will only work while you are at the conference?